pkgs/development/python-modules/semaphore/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, attrs
+, anyio
+}:
+
+buildPythonPackage rec {
+  pname = "semaphore";
```

Comment:
Should pname be the same as Pypi?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

Reply:
Although it has not been updated and will not be in the future, there is a package called semaphore in PyPI.
It is a common word, so I think it is helpful to use `semaphore-bot` to avoid confusion.